### PR TITLE
popover should close on clicking somewhere else

### PR DESCRIPTION
**Pull request by tahaalibra**
Fix: the popover should close when clicking somewhere else.

@Gomez @ChristophWurst

### CODE REVIEW

File: js/views/account.js [hunk 1]
```diff
@@ -65,6 +65,10 @@ define(function(require) {
 		toggleCollapse: function() {
 			this.collapsed = !this.collapsed;
 			this.render();
+		},
+		toggleMenu: function(e) {
+			e.preventDefault();
+			this.menuShown = !this.menuShown;
 			this.listenTo(Radio.ui, 'document:click', function(event) {
```
Comment (ChristophWurst) [inline js/views/account.js]: Hm, doesn't that mean everytime the menu is toggled, another event handler is registered?

Comment (ChristophWurst) [inline js/views/account.js]: add a ``onShow`` method and move that event handler registration in there. That method is called by Marionette once a view has finished rendering and is added to the DOM.

**Comment by tahaalibra** [comment]
@ChristophWurst putting the listening in initialize...does this solve the problem

**Comment by ChristophWurst** [comment]
> @ChristophWurst putting the listening in initialize function...does this solve the problem

Well, did you try the ``onShow`` approach? I'm in favor of that since the initialize method might be called even if the view is not attached to the DOM, which is error-prone…

**Comment by tahaalibra** [comment]
@ChristophWurst using onShow , thanks for help !!...

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 ## 0.5.2 - unreleased
 
+### Fixed
+- Close popover on clicking somewhere else
```
Comment (ChristophWurst) [inline CHANGELOG.md]: awesome :-)

**Comment by ChristophWurst** [comment]
:+1: very nice!

**Comment by ChristophWurst** [comment]
oh, now there are 4 commits. Can you squash them with ``git rebase -i HEAD~4``? Thanks

**Comment by ChristophWurst** [comment]
ah, you're fast :rocket:

**Pull request merged**